src/notes/NLPHashtagPlugin.jsx

```diff
@@ -0,0 +1,290 @@
+import Lang from 'lodash';
+import Collection from 'lodash';
+import Shortcut from '../shortcuts/Shortcut';
+// import Slate from '../lib/slate'
+import { Selection } from '../lib/slate'
+
+const DOMAIN = "http://MM194776-PC.mitre.org"
+const PORT = "8551"
+const API_ROUTE = "/api/parse_sentence"
+const API_ENDPOINT = `${DOMAIN}:${PORT}${API_ROUTE}`
+
+function createOpts(opts) {
+	opts = opts || {};
+	return opts;
+}
+
+function SingleHashtagKeywordStructuredFieldPlugin(opts) {
+    opts = createOpts(opts);
+    const shortcutManager = opts.shortcutManager;
+    const contextManager = opts.contextManager;
+	const createShortcut = opts.createShortcut;
```

Comment:
This is a small code styling thing, but it seems like there is a mix of tabs and spaces in this file.

Reply:
Good catch